extension/src/serialization/collations.rs

```diff
@@ -68,6 +74,11 @@ impl Serialize for PgCollationId {
         S: serde::Serializer,
     {
         unsafe {
+            let mut layout: Option<(&str, &str)> = None;
+            if self.0 == pg_sys::InvalidOid {
```

Comment:
Didn't you just add a function with this very logic?
```suggestion
            if self.is_invalid() {
```

Reply:
Yeah. That code predated the function 😅